docs/docs/guides/authentication/cloudentity.md

```diff
@@ -0,0 +1,110 @@
+# Cloudentity
+
+Kusk Gateway enables using third-party authentication and authorization management solutions such as Cloudentity.
+
+Developers can utilize Cloudentity's platform to control their authorization flows while still using OpenAPI with Kusk to configure gateway level rules.
+
+This separation of concerns is important when your authorization flow complexity grows or the number of services talking to each other require more authorization granularity.
+
+## Setup guide
+
+By the end of this guide, you will have deployed an API to Kusk and used Cloudentity to block requests to your API. 
+
+### 1. Create Kusk Authorizer in Cloudentity
+
+On Cloudentity's platform, go to **Enforcement > Authorizers**, click **CREATE GATEWAY**, and from the list that's displayed choose Kusk.
+
+![cloudentity authorizers list](./img/cloudentity-authorizers.png)
+
+### 2. Install Cloudentity's authorizer in your cluster
+
+The bottom of the quickstart section contains a `helm` command that installs Cloudentity's authorizer and configures with credentials that are used to pull authorization policies from Cloudentity. 
+
+Be sure to click on "Show" before copying the command. 
```

Comment:
Can we copy  those commands  and put them into the guide?

Reply:
I don't think that's possible because it containts `client_id` and `client_secrets`, basically client sensitive information.

Reply:
Cloudentity site is quite complicated and it took me some time to get to the part where the command is.

Especially how to get client_id and other params required for it to run